tests/acceptance/features/webUIUserJourney/journey1.feature

```diff
@@ -25,3 +25,12 @@ Feature: User Journey 1
     And the user restores file "new-lorem2.txt" from the trashbin using the webUI
     And the user browses to the files page
     And file "new-lorem2.txt" should be listed on the webUI
+
+  Scenario: change a file content of a recived shared
+    Given user "Brian" has created file "lorem.txt"
+    And user "Brian" has shared file "lorem.txt" with user "Alice" with "all" permissions
+    And user "Alice" has accepted the share "lorem.txt" offered by user "Brian"
+    When user "Alice" has logged in using the webUI
+    And the user opens folder "Shares" using the webUI
+    When the user uploads overwriting file "lorem.txt" using the webUI
```

Comment:
```suggestion
    And user "Alice" has logged in using the webUI
    When the user opens folder "Shares" using the webUI
    And the user uploads overwriting file "lorem.txt" using the webUI
```

I suggest that you put a single `When` where the "action" starts to really happen.
The steps with "has done something" are intend to be used in the `Given`...`And` section. The grammar sound better in a sentence starting with "Given".

Reply:
```suggestion
    And user "Alice" has logged in using the webUI
    And the user has opened folder "Shares" using the webUI
    When the user uploads overwriting file "lorem.txt" using the webUI
```

This is also a way that the "And" and "When" could be organized. The real "action" step is just the "uploads overwriting".